crates/agglayer-node/src/utils.rs

```diff
@@ -0,0 +1,20 @@
+use url::Url;
+
+pub(crate) fn sanitize_ws_url(url: &str) -> String {
+    let parsed = Url::parse(url);
```

Comment:
Why convert to string and re-parse back to url? Could this just take an `Url` (may require cloning but still avoids the print/parse cycle)